pkg/db/db.go

```diff
@@ -117,10 +118,15 @@ func (c Client) NeedsUpdate(cliVersion string, light, skip bool) (bool, error) {
 
 	}
 
-	if db.SchemaVersion == metadata.Version && metadata.Type == dbType &&
-		c.clock.Now().Before(metadata.NextUpdate) {
-		log.Logger.Debug("DB update was skipped because DB is the latest")
-		return false, nil
+	if db.SchemaVersion == metadata.Version && metadata.Type == dbType {
+		if c.clock.Now().Before(metadata.NextUpdate) {
+			log.Logger.Debug("DB update was skipped because DB is the latest")
+			return false, nil
+		}
+		if c.clock.Now().Before(metadata.DownloadedAt.Add(time.Hour)) {
+			log.Logger.Debug("DB update was skipped because DB was downloaded during the last hour")
+			return false, nil
+		}
```

Comment:
To pass the linter test, we have to define another function like `IsLatestDB` or `IsNewDB`.